superdesk/activity.py

```diff
@@ -121,7 +133,8 @@ class ActivityResource(Resource):
     }
     exclude = {endpoint_name, 'notification'}
     datasource = {
-        'default_sort': [('_created', -1)]
+        'default_sort': [('_created', -1)],
+        'filter': {'_created': {'$gte': utcnow() - datetime.timedelta(days=1)}}
```

Comment:
This filter is applied across all activities but 24hrs filter is specific to Desk Mentions not for other activities.